northstar/src/runtime/state.rs

```diff
@@ -841,4 +850,12 @@ impl<'a> State<'a> {
                 .expect("Internal channel error on main");
         }
     }
+
+    /// Returns an iterator over the containers currently running
+    fn running_containers(&self) -> impl Iterator<Item = &Container> {
+        self.containers
+            .iter()
+            .filter(|(_, state)| state.process.is_some())
```

Comment:
nit: There's a handy [filter_map](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter_map) and I think this should work:
```rust
.filter_map(|(container, state)| state.proces.map(|_| container))
```

Reply:
I wasn't completely happy using either `state.proces.map(|_| container)` or ``state.proces.and(Some(container))`, it reads kind of weird. Fortunately `rust-analyzer` saved the day :tada: and suggested `state.process.is_some().then(|| container)` that is clearer and nicer to read. Didn't know those `.then` and `.then_some` methods on `bool` :)